### DISCUSSION TITLE
Copyleft License

### DISCUSSION THREAD

**Discussion by weex** [WesSouza/MyPub#2] [General]
I posted as a comment to https://github.com/WesSouza/MyPub/discussions/1 but per your advice I'm putting this suggestion in it's own thread.

If you're looking to build a community around this please choose a copyleft license. Non-copyleft is a non-starter for me because it enables proprietary competition that can always be a step ahead of the community. In essence you force your community to compete with the proprietary while helping the proprietary entity at the same time. I believe this is a subtle turn off for many other OSS contributors.

You requested more info and for that I offer this post: https://socialhub.activitypub.rocks/t/best-license-for-open-source-fedi-apps/2876/4

The book I reference is a great read as well.

-David

**Comment by qertis** [comment]
Try to use GPL-3
